react/components/ShippingSimulator/Loader.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import ContentLoader from 'react-content-loader'
+
+import shippingSimulator from './shippingSimulator.css'
```

Comment:
nit:
```suggestion
import styles from './shippingSimulator.css'
```

Reply:
If I name *styles* it gets conflicted with the props @lucasecdb

Reply:
the prop is on the `ShippingSimulator`, no?